src/opds_dumper.cpp

```diff
@@ -168,6 +79,32 @@ string OPDSDumper::dumpOPDSFeedV2(const std::vector<std::string>& bookIds, const
     });
   }
 
+  return bookData;
+}
+
+} // unnamed namespace
+
+string OPDSDumper::dumpOPDSFeed(const std::vector<std::string>& bookIds, const std::string& query)
+{
+  const auto bookData = getBookData(library, bookIds);
+  const kainjow::mustache::object template_data{
+     {"date", gen_date_str()},
+     {"root", rootLocation},
+     {"feed_id", gen_uuid(libraryId + "/catalog/search?"+query)},
+     {"filter", query.empty() ? MustacheData(false) : MustacheData(query)},
+     {"totalResults", to_string(m_totalResults)},
+     {"startIndex", to_string(m_startIndex)},
+     {"itemsPerPage", to_string(m_count)},
+     {"books", bookData }
+  };
```

Comment:
It seems that this data is pretty closed from the ones used in `dumpOPDSFeedV2`.
Could be have "only one method" using two different templates ?
(Or maybe even better, one template with just the endpoint changing ? If possible)

Reply:
Let's consider that after the new API is finalized. I don't want the old API to be a drag on the new API, and a premature attempt to share code between them will do just that.